app/src/main/java/fi/helsinki/opintoni/integration/newsfeeds/AtomRestClient.java

```diff
@@ -24,7 +24,7 @@
 import org.springframework.web.client.RestTemplate;
 
 public class AtomRestClient {
-    private final static Logger log = LoggerFactory.getLogger(AtomRestClient.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(AtomRestClient.class);
```

Comment:
SLF4J-loggerin tapauksessa lowercase-nimeäminen on yleisesti "oikeampaa" kuin yleinen uppercase-constant-nimeäminen, ks. esim. https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case#answer-1417216.

Perusteluna tässä siis se että `static final`in pitää olla uppercase vain kun se on vakio, mikä käytännössä tarkoittaa immutable `static final`ia, mitä slf4j-loggerin ei voi ajatella olevan kun se on objekti jolla on lisäksi metodeja joilla on sivuvaikutuksia.

Tuon poikkeuksen saa ilmeisesti konffattua checkstyleen näin:
```
<module name="ConstantName">
    <property name="format"
          value="^log(ger)?|[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$"/>
</module>
```

Reply:
Hyvää tietoa!
Lisäsin tuon, ja muutin projektista muut LOGGER -tapaukset projektissa paikoitellen käytettyyn log -versioon
(oli 50/50 LOGGER ja log)

Reply:
Tämänkin säännön voisi lisätä UniHow:n survey-servicelle (ei varmaan nyt ole)